cmd/goignore/main.go

```diff
@@ -48,17 +48,6 @@ var newCmd = &cobra.Command{
 			}
 		}
 
-		// Check if .git repo exists, if not initialize it
```

Comment:
instead of removing this, just comment it out because as a future feature, I want to give users the option to choose between having a git repo initialize or not

Reply:
Alright, makes sense

Reply:
Pushed changes